src/ManageCourses.Api/Middleware/BearerTokenApiKeyOptions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Globalization;
+using Microsoft.AspNetCore.Authentication;
+
+namespace GovUk.Education.ManageCourses.Api.Middleware
+{
+    public class BearerTokenApiKeyOptions : AuthenticationSchemeOptions    
+    {
+        public string ApiKey { get; set; }
+
+        public override void Validate()
+        {
+            if (string.IsNullOrWhiteSpace(this.ApiKey)) {
+                throw new ArgumentException(string.Format(CultureInfo.CurrentCulture, "The '{0}' option must be provided.", nameof(this.ApiKey)), nameof(this.ApiKey));
```

Comment:
Is the culture info necessary?

Reply:
Origin
https://github.com/aspnet/Security/blob/442df16e265ab1955d7b8468cf1d8b38b8d915c7/src/Microsoft.AspNetCore.Authentication.OAuth/OAuthOptions.cs#L33

Reply:
I'm not sure that answers my question other than implying it was a copy paste, but I don't mind if this goes in as is.